controllers/space/space_controller_test.go

```diff
@@ -716,7 +706,7 @@ func TestDeleteSpace(t *testing.T) {
 			spacetest.AssertThatSpace(t, s.Namespace, s.Name, hostClient).
 				DoesNotExist()
 			AssertThatCountersAndMetrics(t).
-				HaveSpacesForCluster("member-1", 0) // space counter is decremented when there are no finalizers on the space resource that is being deleted
+				HaveSpacesForCluster("member-1", 1) // space counter is not decremented
```

Comment:
probably I'm missing something - but how can the counter not be decremented when Space is deleted ?

Reply:
because this is the last step when the NSTemplateSet is gone and we just need to remove the finalizer.
We count the representation of the Spaces in the member clusters - the representation is NSTemplateSet, so we need to count when we create or delete it

Reply:
ok , so in this case the counter was already decremented , the test now mainly ensures that the finalizer is correctly removed from the Space. 
I was just a bit confused by seeing the sapce counter initialized with 1 and not decremented.